examples/fscloud/variables.tf

```diff
@@ -42,3 +42,8 @@ variable "allowlist" {
   default     = []
   description = "Set of IP address and description to allowlist in database"
 }
+
+variable "existing_hpcs_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto service in which the key specified in var.hpcs_key_crn is coming from"
```

Comment:
I don't see any variable named `hpcs_key_crn` in here?

Reply:
It's there, but it was marked optional with a default value of "null". I think that's a mandatory field. Hence updated.

Reply:
Apologies, I thought you were talking about `hpcs_key_crn` in the profiles/fscloud/variables.tf. I misread. Added here.